packages/ckeditor5-dev-ci/bin/trigger-circle-build.js

```diff
@@ -5,43 +5,37 @@
  * For licensing, see LICENSE.md.
  */
 
+import { getArgument } from '../lib/parse-argument.js';
 import triggerCircleBuild from '../lib/trigger-circle-build.js';
 
 /**
  * This script triggers a new CircleCI build.
  *
- * In order to integrate the action in your pipeline, you need prepare a few environment variables:
+ * In order to integrate the action in your pipeline, you need prepare a few CLI or environment variables:
  *
- *   - CIRCLE_BRANCH - provided by default by CircleCI and keeps the git branch of processed build.
- *   - CKE5_COMMIT_SHA1 - a full commit identifier of the processed the build.
- *   - CKE5_CIRCLE_TOKEN - an authorization token to talk to CircleCI REST API.
- *   - CKE5_GITHUB_REPOSITORY_SLUG - a repository slug (org/name) where a new build will be started.
- *   - CKE5_GITHUB_RELEASE_BRANCH - (optional) define a branch that leads the release process.
- *   - CKE5_GITHUB_TRIGGER_REPOSITORY_SLUG - (optional) a repository slug (org/name) that triggers a new build.
+ *   - `circle-branch` - provided by default by CircleCI and keeps the git branch of processed build.
+ *   - `cke5-commit-sha1` - a full commit identifier of the processed the build.
+ *   - `cke5-circle-token` - an authorization token to talk to CircleCI REST API.
+ *   - `cke5-github-repository-slug` - a repository slug (org/name) where a new build will be started.
+ *   - `cke5-github-release-branch` - (optional) define a branch that leads the release process.
+ *   - `cke5-github-trigger-repository-slug` - (optional) a repository slug (org/name) that triggers a new build.
  *
  * Example usage:
- * CKE5_CIRCLE_TOKEN=... ckeditor5-dev-ci-trigger-circle-build
+ * ckeditor5-dev-ci-trigger-circle-build --cke5-circle-token=...
  */
 
-const {
-	CKE5_COMMIT_SHA1,
-	CIRCLE_BRANCH,
-	CKE5_CIRCLE_TOKEN,
-	CKE5_GITHUB_RELEASE_BRANCH,
-	CKE5_GITHUB_REPOSITORY_SLUG,
-	CKE5_GITHUB_TRIGGER_REPOSITORY_SLUG
-} = process.env;
-
 const options = {
-	circleToken: CKE5_CIRCLE_TOKEN,
-	commit: CKE5_COMMIT_SHA1,
-	branch: CIRCLE_BRANCH,
-	releaseBranch: CKE5_GITHUB_RELEASE_BRANCH,
-	repositorySlug: CKE5_GITHUB_REPOSITORY_SLUG
+	circleToken: getArgument( 'cke5-circle-token' ),
```

Comment:
```suggestion
	circleToken: getArgument( 'circle-token' ),
```

or even

```suggestion
	circleToken: getArgument( 'token' ),
```

I can't imagine any use case that would pass two different tokens here.